plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -175,6 +175,6 @@ builtinMemoryModels = BuiltinCostModelBase
   , paramExpModInteger                   = Id $ ModelThreeArgumentsConstantCost 100000000000 -- FIXME: stub
   -- paramCaseList
   -- paramCaseData
-  , paramDropList                        = Id $ ModelTwoArgumentsLinearInY $ OneVariableLinearFunction 4 0
+  , paramDropList                        = Id $ ModelTwoArgumentsConstantCost 4
```

Comment:
That's 32 bytes.  Are we happy with that?  I'm not sure how much room a pointer takes up, but it's probably 8 bytes at most.  I don't mind all that much though since we're a bit inconsistent about this anyway: many of the builtins have a constant memory cost of 32 `ExMemory` (256 bytes).

Reply:
> That's 32 bytes. Are we happy with that?

I have absolutely no idea.